src/test/java/com/mongodb/jdbc/MongoSQLMock.java

```diff
@@ -187,7 +247,56 @@ protected static MongoJsonSchema generateMongoJsonSchema() {
     }
 
     BsonDocument generateRow() {
-        /**/
-        return null;
+        /*
+        {
+            "foo.a":1,
+            "foo.b":null,
+            "foo.c":2,
+            "foo.d":{
+                "$undefined":true
+            },
+             "foo.null": null
+             "foo.vec":[
+                1,
+                2,
+                3
+            ],
+            "__bot.a":1.2,
+            "__bot.str":"a"
```

Comment:
```suggestion
            "__bot.a":1.2,
            "__bot.binary": <binary data>
            "__bot.str":"a"
```
update comment to include new field. I don't think we need to write out the extended json representation of the data; the fact that it is binary is the important part, not the specific value.